commander/src/bootstrapping/commands/config/create.ts

```diff
@@ -17,24 +17,22 @@ import { Command, flags as flagParser } from '@oclif/command';
 import fs from 'fs-extra';
 import { join, resolve } from 'path';
 import inquirer from 'inquirer';
-import { defaultConfig } from '../../utils/config';
+import { defaultConfig } from '../../../utils/config';
 
-export default class NetworkConfigCommand extends Command {
+export class CreateCommand extends Command {
 	static description = 'Creates network configuration file.';
 	static examples = [
-		'generate:network-config mydir',
-		'generate:network-config mydir --label alpha-sdk-app',
-		'generate:network-config mydir --label alpha-sdk-app --community-identifier sdk',
-	];
-	static args = [
-		{
-			name: 'networkName',
-			description: 'Directory where the config file is saved.',
-			required: true,
-		},
+		'generate:network-config --network-name mydir',
+		'generate:network-config --network-name mydir --label alpha-sdk-app',
+		'generate:network-config --network-name mydir --label alpha-sdk-app --community-identifier sdk',
 	];
 
 	static flags = {
+		'network-name': flagParser.string({
```

Comment:
Should not we use `--output`, `--output-dir` rather network name here? as its just pointing to any directory, which have no link with term network.